File: IssueTracker_210042122/src/Comment.java
import java.sql.Date;

public class Comment {
    private int id;
    private String content;

    private String Author;

    private Date createdAt;

    public Comment(int id, String content, String author, Date createdAt){
        this.id=id;
        this.Author=author;
        this.content=content;
        this.createdAt=createdAt;
    }
}

File: TaskManager/TaskManager_210042112/TaskManager_112/TaskManager_112/Issues/Comment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TaskManager_112
{
    public class Comment
    {
        public int Id { get; set; }
        public string Content { get; set; }
        public string Author { get; set; }
        public DateTime CreatedAt { get; set; }
        public Comment(int id, string content, string author)
        {
            Id = id;
            Content = content;
            Author = author;
            CreatedAt = DateTime.Now;

        }
    }
}
